src/promdens/promdens.py

```diff
@@ -804,73 +829,61 @@ def parse_cmd_args():
     parser.add_argument("-p", "--plot", action="store_true", help="Plot the input data and calculated results and save them as png images.")
     parser.add_argument("input_file", help="Input file name.")
 
-    return vars(parser.parse_args())
+    return parser.parse_args()
 
 
 def main():
     # Parse the command line parameters and print them
     config = parse_cmd_args()
     print_header()
     print("* Input parameters:")
-    for item in config:
+    for key, value in vars(config).items():
         add = ''
-        if item == 'nsamples' and config[item] == 0:
+        if key == 'nsamples' and value == 0:
             add = '(All input data will be used)'
-        if config['method'] == 'pdaw' and item in ('npsamples', 'random_seed', 'preselect', 'neg_handling'):
+        if config.method == 'pdaw' and key in ('npsamples', 'random_seed', 'preselect', 'neg_handling'):
             continue
-        print(f"  - {item:20s}: {config[item]}   {add}")
-    method = config['method']
-    nsamples = config['nsamples']
-    new_nsamples = config['npsamples']
-    nstates = config['nstates']
-    plotting = config['plot']
-    energy_unit = config['energy_unit']
```

Comment:
I found these reassignments rather superfluous at this point. I think we can do without them.